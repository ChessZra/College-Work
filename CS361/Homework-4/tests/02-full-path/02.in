/bin/cat tests/02-full-path/file1.txt
