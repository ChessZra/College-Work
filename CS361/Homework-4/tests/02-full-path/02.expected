cs361sh> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum ut lobortis urna, vel semper orci. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Nunc id purus orci. Integer tellus massa, tempor at tincidunt a, efficitur sed lacus. Vivamus vulputate tellus eget semper fermentum. In non arcu porta, sagittis arcu in, interdum justo. Suspendisse et auctor enim. In metus felis, elementum non dolor eu, porta pellentesque urna. Vestibulum tincidunt est augue. In id dolor luctus, rutrum libero quis, blandit odio. Aenean ac justo pretium, euismod dolor sagittis, laoreet lectus. Cras accumsan dolor odio, vitae cursus felis elementum ut. Nulla in nisi et dui ornare viverra ut pulvinar nibh. Nam consequat erat non justo vulputate, eu egestas justo ornare.
Morbi lorem leo, egestas et nibh non, feugiat iaculis enim. Nulla quis tortor commodo nibh vulputate faucibus eget at est. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris sed diam lacinia, maximus massa vitae, pellentesque libero. Interdum et malesuada fames ac ante ipsum primis in faucibus. Aenean id nunc sem. Mauris dignissim elit sit amet sem condimentum, id euismod ipsum tristique. Aliquam quis tristique urna. Suspendisse nec sem nibh. Proin tincidunt aliquam ligula, sed sollicitudin lectus suscipit eget. Fusce efficitur aliquet tortor. Sed consectetur dui velit, vel interdum nisl imperdiet in. Donec sed augue ipsum. In dignissim blandit erat a elementum. Nam pellentesque sem sed massa placerat placerat.
Praesent mi eros, euismod ac varius ac, suscipit nec neque. Morbi ex justo, pharetra in scelerisque nec, consectetur id neque. Proin vitae mauris iaculis, volutpat magna et, pellentesque felis. Cras dui ex, fermentum fringilla massa ullamcorper, varius consectetur nisl. Phasellus in leo ex. Donec cursus finibus nisl, a lacinia ex eleifend a. Maecenas pretium pretium mollis. Phasellus vestibulum ipsum sed erat dapibus sagittis. Nulla feugiat magna lectus, vel varius nulla aliquam ut. Integer eget nisl sem. Quisque a lorem elementum, eleifend magna vel, facilisis neque. Ut auctor ante et mauris dignissim, eget finibus arcu tempor. Nunc nec sagittis ipsum.
Praesent iaculis condimentum nibh, sit amet vehicula odio tristique in. Vestibulum eu metus pharetra, viverra quam et, dignissim orci. Sed vitae finibus quam. Morbi eget tortor ac sem faucibus ullamcorper vitae at nulla. Praesent viverra convallis urna, nec placerat neque vehicula at. Ut rutrum est nunc, vel tempor diam vehicula ut. Nulla sed cursus neque, quis dignissim quam. Vivamus a felis at sapien tincidunt consectetur eget sit amet mauris. Duis volutpat eu ligula in efficitur.
Praesent iaculis, justo eget ullamcorper dapibus, lorem sem faucibus metus, sit amet fringilla felis massa eu mauris. Suspendisse mauris arcu, aliquet tempor placerat a, feugiat pulvinar elit. Praesent laoreet suscipit rhoncus. Nunc finibus, mi vitae finibus porttitor, sapien enim egestas ipsum, at dapibus risus dui tempor risus. Donec mattis mauris ac urna iaculis luctus. Vivamus imperdiet elit in lorem ultricies tincidunt. Etiam quis ante ut urna hendrerit ultrices aliquet quis arcu.
Phasellus semper urna fermentum lectus convallis efficitur. Nunc a justo accumsan nulla imperdiet malesuada vitae a arcu. Maecenas fermentum ultrices metus, sed tempus ante auctor.cs361sh> 
